src/utilities.ts

```diff
@@ -382,6 +409,51 @@ export async function computeOptionChainAddress(
     return optionChainAddress
 }
 
+/**
+ * Compute address of on-chain short aggregator contract using CREATE2 functionality.
+ *
+ * @param ticker Ticker of the underlying asset.
+ * @param version Version of Arrow contract suite with which to interact. Default is V4.
+ * @param wallet Wallet with which you want to connect the instance of the Arrow registry contract. Default is Fuji provider.
+ * @returns Address of the short aggregator corresponding to the passed ticker.
+ */
+export async function computeShortAggregatorAddress(
+    ticker: Ticker,
+    version = DEFAULT_VERSION,
+    wallet:
+        | ethers.providers.Provider
+        | ethers.Wallet
+        | ethers.Signer = providers.fuji
+): Promise<string> {
+    // Get chain factory contract address from router
+    const router = getRouterContract(version, wallet)
+    
+    let shortAggregatorFactoryAddress = undefined
+    switch (version) {
+        case Version.V4:
+        case Version.V3:
+        case Version.COMPETITION:
+            shortAggregatorFactoryAddress = await router.getShortAggregatorFactoryAddress()
```

Comment:
It seems like we don't need this `switch` statement at all. We could just write `const shortAggregatorFactoryAddress = await router.getShortAggregatorFactoryAddress()`